recipes/libpq/meson/conanfile.py

```diff
@@ -0,0 +1,202 @@
+        if self.settings_build.os == "Windows":
+            self.tool_requires("winflexbison/2.5.25")
+        else:
+            self.tool_requires("flex/2.6.4")
+            self.tool_requires("bison/3.8.2")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        def feature(v):
+            return "enabled" if v else "disabled"
+
+        tc = MesonToolchain(self)
+        tc.project_options["ssl"] = "openssl" if self.options.with_openssl else "disabled"
+        tc.project_options["icu"] = feature(self.options.with_icu)
+        # Why did the old version disable this explicitly?
+        tc.project_options["zlib"] = feature(self.options.with_zlib)
+        tc.project_options["zstd"] = feature(self.options.with_zstd)
+        tc.project_options["libxml"] = feature(self.options.with_libxml2)
+        tc.project_options["lz4"] = feature(self.options.with_lz4)
+        tc.project_options["libxslt"] = feature(self.options.with_xslt)
+        tc.project_options["readline"] = feature(self.options.get_safe("with_readline"))
+
+        tc.project_options["ldap"] = "disabled"
+        tc.project_options["tap_tests"] = "disabled"
+        tc.project_options["plpython"] = "disabled"
+        tc.project_options["docs"] = "disabled"
+        tc.generate()
+        deps = PkgConfigDeps(self)
+        deps.generate()
+
+    def build(self):
+        meson = Meson(self)
+        meson.configure()
+        meson.build()
+
+    def package(self):
+        copy(self, "COPYRIGHT", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        meson = Meson(self)
+        meson.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.pdb", self.package_folder, recursive=True)
+
+        # INFO: When building on Windows:
+        # - MSVC + Static: produces .a files
+        # - MSVC + Shared: produces .lib + .dll files
+        # - MinGW + Static: produces .a files
+        # - MinGW + Shared: produces .dll.a + .dll files
+        if self.options.shared:
+            rm(self, "*.a", os.path.join(self.package_folder, "lib"), excludes="*.dll.a")
+        else:
+            rm(self, "*.so*", os.path.join(self.package_folder, "lib"))
+            rm(self, "*.dylib", os.path.join(self.package_folder, "lib"))
+            rm(self, "*.dll", os.path.join(self.package_folder, "bin"))
+            rm(self, "*.lib", os.path.join(self.package_folder, "lib"))
+            rm(self, "*.dll.a", os.path.join(self.package_folder, "lib"))
+        fix_apple_shared_install_name(self)
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "PostgreSQL")
+        # INFO: Upstream libpq.pc is specific for the main library, not the whole package.
+        self.cpp_info.set_property("pkg_config_name", "__libpq")
```

Comment:
can we set this to `none` as of 2.18?
https://github.com/conan-io/conan/pull/18439

Reply:
Good catch!